lib/task-data/schemas/dell-wsman-get-config-catalog.json

```diff
@@ -0,0 +1,81 @@
+{
+    "copyright": "Copyright 2017 Dell Inc. or its subsidiaries. All Rights Reserved.",
+    "definitions": {
+        "serverIP": {
+            "description": "Dell server iDARC IP address",
+            "type": "string"
```

Comment:
the serverIP should not be empty, so add more constraint (such as `minLength` or even `pattern`) for it will help user to identify problem earlier. The same for others.

Reply:
Good idea!